lib/src/sunnah_times.dart

```diff
@@ -4,14 +4,14 @@ import 'data/date_components.dart';
 
 class SunnahTimes {
   /// The midpoint between Maghrib and Fajr
-  DateTime _middleOfTheNight;
+  late DateTime _middleOfTheNight;
```

Comment:
Same _middleOfTheNight can be null as prayerTimes.magrib can be null.

Reply:
the last commit makes sure it can't.